README.md

```diff
@@ -1,6 +1,53 @@
-# curriculum-cr-training-mobile-menu
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
+# Mobile Menu
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+> This repo host an initial code for the tasks performed during the Code Reviewers training.
+> Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+### Screenshot
+
+![screenshot](./app_screenshot.png)
+
+## Live Demo
+[Live Demo Link](https://shadyshawkat.github.io/Mobile-Menu-CR/)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
```

Comment:
Please make sure that you `getting started` section in your readme file is filled by write information.
This section is important to help recruiters (not technical people) to show them how to run and see your code.
 You can find the general requirements [here](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/blob/master/general_requirements.md). if any is missing, the project will be invalid.